dicom_utils/types.py

```diff
@@ -538,14 +538,23 @@ def short_str(self) -> str:
             return ""
         return self.name.lower()
 
+    @property
+    def is_standard_view(self) -> bool:
+        return self in {self.CC, self.MLO}
+
+    @property
+    def is_mlo_like(self) -> bool:
+        return self in {self.MLO, self.ML, self.LMO, self.LM}
+
+    @property
+    def is_cc_like(self) -> bool:
+        return self in {self.CC, self.XCCL, self.XCCM}
+
 
 class MammogramView(NamedTuple):
     laterality: Laterality = Laterality.UNKNOWN
     view: ViewPosition = ViewPosition.UNKNOWN
 
-    def __repr__(self):
```

Comment:
Just curious why you decided to remove this

Reply:
We can consider restoring this. There were a few issues that prompted me to remove it. It has no test coverage, and the function fails if attributes aren't assigned correctly. I had a failing test where `None` was being placed into an attribute, but I couldn't get a `repr` to debug the problem because the current implementation has some assumptions about correct input types.

If this implementation isn't needed I'd rather let `super()` handle the repr, though we could have a try/except to fallback when needed.